indico/modules/events/registration/templates/display/regform_list.html

```diff
@@ -44,7 +44,7 @@
                     {% if any_published_registrations %}
                         {% if regform.publish_registration_count %}
                             <td class="icon-user">
-                                {{- regform.active_registration_count -}}
+                                {{- regform.existing_registrations_count -}}
```

Comment:
I'm not sure if this one should be changed. Why would anyone but a manager care about a rejected registration? Such a person is not participating in the event and also not counting against the limit for example!

Reply:
Yeah, I wasn't really sure about this one. Updated.